File: app/src/main/java/cn/intret/app/picgo/ui/event/ImageFragmentSelectionChangeMessage.java
package cn.intret.app.picgo.ui.event;


public class ImageFragmentSelectionChangeMessage {
    int currentCode;

    String transitionName;

    public String getTransitionName() {
        return transitionName;
    }

    public ImageFragmentSelectionChangeMessage setTransitionName(String transitionName) {
        this.transitionName = transitionName;
        return this;
    }

    public int getCurrentCode() {
        return currentCode;
    }

    public ImageFragmentSelectionChangeMessage setCurrentCode(int currentCode) {
        this.currentCode = currentCode;
        return this;
    }
}

File: app/src/main/java/cn/intret/app/picgo/ui/event/ImageFragmentSelectionChangeMessage.kt
package cn.intret.app.picgo.ui.event


class ImageFragmentSelectionChangeMessage {
    internal var currentCode: Int = 0

    internal lateinit var transitionName: String

    fun getTransitionName(): String {
        return transitionName
    }

    fun setTransitionName(transitionName: String): ImageFragmentSelectionChangeMessage {
        this.transitionName = transitionName
        return this
    }

    fun getCurrentCode(): Int {
        return currentCode
    }

    fun setCurrentCode(currentCode: Int): ImageFragmentSelectionChangeMessage {
        this.currentCode = currentCode
        return this
    }
}
